### PR TITLE
added function and bdd for delete wishlist

### PR DESCRIPTION
**Pull request by raymondlu8547**
added function and bdd for delete wishlist

### CODE REVIEW

File: features/wishlists.feature [hunk 1]
```diff
@@ -27,3 +27,11 @@ Scenario: Create a Wishlist
     And I should see "Beckett's Halloween Costume List" in the "Name" field
     And I should see "42" in the "Customer ID" field
     And "Is Public" should "be" checked
+
+Scenario: Delete a Wishlist
+    When I visit the "Admin UI Page"
```
Comment (jay-goettelmann-nyu) [inline features/wishlists.feature:32]: Hey @raymondlu8547, good start but a couple things to update.  #1 I think we will need to create a wishlist first before we test the delete.  We designed our wishlist API to return a 204 No Content response when deleting a non-existent wishlist_id.  (Which is what happens in this case).  In the $.ajax().done() method we should check the response code and perhaps give a different message for each.  Like "Wishlist with ID [5432] successfully deleted" for a 200 response and maybe "Could not find a wishlist to delete with ID [42]" when the response is 204.

So in your behave test you should:
1. Create a wishlist
2. Read the new ID 
3. Try to delete the newly created ID
4. Assert that the message is the 200 success message
5. Try to delete the same ID a second time
6. Assert that the message is the 204 message

File: service/static/js/rest_api.js [hunk 2]
```diff
@@ -131,25 +131,25 @@ $(function () {
...
-            url: `/pets/${pet_id}`,
+            url: `/wishlists/${wishlist_id}`,
             contentType: "application/json",
             data: '',
         })
 
         ajax.done(function(res){
             clear_form_data()
-            flash_message("Pet has been Deleted!")
+            flash_message("Successfully deleted Wishlist with ID " + wishlist_id)
```
Comment (jay-goettelmann-nyu) [inline service/static/js/rest_api.js:152]: Here you can add a conditional to check for res.status == 204 to set the ID not found message and you can use the else clause to assume it was a successful deletion.

**Review by jay-goettelmann-nyu — CHANGES_REQUESTED** [review]
Hey Raymond.  Good start.  Please see my comments for a couple needed changes.  Thansk!

**Review by jay-goettelmann-nyu — APPROVED** [review]
I added a commit to address a couple things.  I will approve but we should have others review as well.

**Review by rahilkadakia — APPROVED** [review]
Looks good. Merging.

**Pull request merged**